.github/workflows/ci.yml

```diff
@@ -429,6 +430,15 @@ jobs:
       - run: pytest modin/pandas/test/test_io.py::TestCsv --verbose
       - run: pytest modin/test/interchange/dataframe_protocol/test_general.py
       - run: pytest modin/test/interchange/dataframe_protocol/hdk
+      - run: python examples/docker/modin-hdk/census-hdk.py examples/data/census_1k.csv -no-ml
+      - run: python examples/docker/modin-hdk/nyc-taxi-hdk.py examples/data/nyc-taxi_1k.csv
+      - run: |
+          python examples/docker/modin-hdk/plasticc-hdk.py \
+          examples/data/plasticc_training_set_1k.csv \
+          examples/data/plasticc_test_set_1k.csv \
+          examples/data/plasticc_training_set_metadata_1k.csv \
+          examples/data/plasticc_test_set_metadata_1k.csv \
+          -no-ml
```

Comment:
How much memory does the data files occupy?

Reply:
About 90 Mb.

Reply:
Not sure this is fine. Can we use datasets from a remote bucket?

cc @modin-project/modin-core, @modin-project/modin-contributors

Reply:
Seems there is a misunderstanding. It consumes about 90 Mb in runtime, when the files are loaded into memory, but the dataset files are not so big, I've truncated the original files to 1000 lines:
36K     examples/data/plasticc_test_set_1k.csv
84K     examples/data/plasticc_test_set_metadata_1k.csv
40K     examples/data/plasticc_training_set_1k.csv
88K     examples/data/plasticc_training_set_metadata_1k.csv